scripts/hostmap-crtsh.nse

```diff
@@ -14,43 +14,20 @@ References:
 ---
 -- @args hostmap.prefix If set, saves the output for each host in a file
 -- called "<prefix><target>". The file contains one entry per line.
+--
+-- @args hostmap-crtsh.strict If set (default true), only return *real* subdomains.
+-- If strict=false, returns all identities containing the hostname (old behavior).
+--
 -- @args newtargets If set, add the new hostnames to the scanning queue.
--- This the names presumably resolve to the same IP address as the
--- original target, this is only useful for services such as HTTP that
--- can change their behavior based on hostname.
+-- This is useful for services that change their behavior based on hostname.
 --
 -- @usage
--- nmap --script hostmap-crtsh --script-args 'hostmap-crtsh.prefix=hostmap-' <targets>
+-- nmap --script hostmap-crtsh --script-args 'hostmap-crtsh.strict=true' <targets>
 -- @usage
--- nmap -sn --script hostmap-crtsh <target>
--- @output
```

Comment:
Do not delete the `@output` section. It has its purpose. Read the documentation for details.